pitop/pma/adc_base.py

```diff
@@ -22,6 +23,12 @@ class ADCBase(Stateful, Recreatable):
     """
 
     def __init__(self, port_name, pin_number=1, name="adcbase"):
+        if port_name not in Port.keys():
+            raise ValueError(f"{port_name} is not a valid port name. An example of a valid port name is A0")
```

Comment:
It's usually better to create a new Exception class specific for things like this, but we don't really have much in the way of uniform exception handling. Perhaps this should be created as a new Issue for across the SDK rather than tackling that here.